mne/tests/test_epochs.py

```diff
@@ -1302,6 +1302,8 @@ def test_epoch_eq():
     cond1, cond2 = ['a', ['b', 'b/y']], [['a/x', 'a/y'], 'x']
     for c in (cond1, cond2):  # error b/c tag and id mix/non-orthogonal tags
         assert_raises(ValueError, epochs.equalize_event_counts, c)
+    assert_raises(ValueError, epochs.equalize_event_counts,
+                  ["a/no_match", "b"])
```

Comment:
Wondering if one could simply be a KeyError, while the other remains a ValueError.